adage/analyze/api.py

```diff
@@ -342,7 +350,7 @@ def apply_filters(self, request, applicable_filters):
 
 class ParticipationResource(ModelResource):
     node = fields.IntegerField(attribute='node_id', null=False)
```

Comment:
Good point. The key relationships are captured and enforced at the level of `models.py` but not here in `api.py`. If the attributes are explicitly declared to TastyPie as `ForeignKey`s, we get the ability to access those related objects "for free."

The only reason I can see not to do this is if we don't want to expose those models directly via an API, but I think there is no harm in doing that. Shouldn't we make all of our relationships explicit via TastyPie?